tools/CI.jenkinsfile

```diff
@@ -71,6 +71,13 @@ pipeline
         }
       }
     }
+    stage('Test+AddressSanitizer')
+    {
+      steps
+      {
+        sh 'SANITIZER="address undefined leak" make test'
```

Comment:
I was a bit suspicious that the CI server didn't fail on this, because I know the address sanitizer reports a problem!

Two updates here:

1. Please split "address" and "undefined" into two separate test cases. "leak" (I'm guessing that's actually `memory`?) can be removed, it's not currently supported on the compilers I have (I need to look into that).
1. For each of the sanitizer test cases, please add a `BUILDRESULTS` variable override that places the ouptut somewhere inside of the default `buildresults` folder (example: `BUILDRESULTS=buildresults/address-sanitizer`). This will ensure that the project is properly reconfigured.

Also... I'm sorry. I feel bad for the multiple change requests when I totally could have caught this the first time if I actually looked.

Reply:
It's `-fsanitize=leak` : https://gcc.gnu.org/onlinedocs/gcc/Instrumentation-Options.html#index-fsanitize_003dleak
Not sure exactly the meson incantation to get it through. I'll separate out each sanitizer (though IMO it's better to flip them all on, all the time :grinning: ).

Reply:
Ah, I think that Meson doesn't support the leak sanitizer (yet), but that should be a simple change to the option/compiler flags. I'll look into making that change.